hasher-matcher-actioner/webapp/src/forms/PreviewableDropzone.tsx

```diff
@@ -39,13 +40,14 @@ function PreviewPane({type, handleChange, file}: PreviewPaneProps) {
         </Col>
       </Row>
       <Row>
-        <Col
-          style={{textAlign: 'center', maxHeight: '300px', overflow: 'scroll'}}>
-          {type === 'video' ? (
-            <BlurVideo src={URL.createObjectURL(file)} override={!revealed} />
-          ) : (
-            <BlurImage src={URL.createObjectURL(file)} override={!revealed} />
-          )}
+        <Col>
+          <ResponsiveEmbed aspectRatio="16by9">
+            {type === 'video' ? (
```

Comment:
why not just do this https://github.com/facebook/ThreatExchange/pull/805/files#diff-557a7cfcc6b8260e93c3cea9b7f62ea2a4ac0dedb89b129ffa7924df6c6cc005R79?

Reply:
Because type is of type `FileType` defined above.

Not using ContentType because we may (some day far in the future) use images for previews of videos. So, content_type != preview.file_type.

Reply:
On second thought, we are not doing that now, so vote for consistency. I'll make the change, use ContentType through out.